### PR TITLE
GRAPHICS-1

### PR DESCRIPTION
**Pull request by thekingofjibs**
Partially there so far, trying to unpack the form lifecycle that's been setup - for some reason number inputs refuse to work as seamlessly as any other input and I'm a little stumped as to what's different from the surface? I can go digging into source but I figured flagging it incase I'd missed anything stupid was worth a punt!

### CODE REVIEW

File: scores-src/src/common/components/Clock.tsx [hunk 1]
```diff
@@ -23,8 +24,6 @@ export function RenderClock({
     return () => clearInterval(interval);
   }, [clock.state]);
   return (
-    <span style={{ fontVariantNumeric: "tabular-nums" }}>
-      {formatMMSSMS(time, precisionMs, precisionHigh)}
-    </span>
+    <Title order={3}>{formatMMSSMS(time, precisionMs, precisionHigh)}</Title>
```
Comment (markspolakovs) [inline scores-src/src/common/components/Clock.tsx:27]: You'll want to preserve the (JSX equivalent of the) CSS `font-variant-numeric: tabular-nums` - that ensures the numbers keep a consistent letter spacing and avoids it jumping around when it changes.

Comment (thekingofjibs) [reply scores-src/src/common/components/Clock.tsx]: oh, good call - amended and pushed

File: scores-src/src/client/pages/CreateEventModal.tsx [hunk 2]
```diff
@@ -32,48 +26,47 @@ export function CreateEventModal() {
...
-                <Alert variant="danger">Could not create! {submitError}</Alert>
-              )}
-              {import.meta.env.DEV && <code>{JSON.stringify(errors)}</code>}
-            </BootstrapForm>
-          )}
-        </Formik>
-      </Modal.Body>
+    <Modal opened onClose={() => nav("..")}>
+      <Title>Create Event</Title>
+      <Stack>
```
Comment (markspolakovs) [inline scores-src/src/client/pages/CreateEventModal.tsx:31]: NITPICKING: why's this in a stack of its own?

File: scores-src/src/client/pages/CreateEventModal.tsx [hunk 3]
```diff
@@ -32,48 +26,47 @@ export function CreateEventModal() {
...
+      </Stack>
+      <Formik
+        enableReinitialize
+        initialValues={{}}
+        onSubmit={submit}
+        validationSchema={EVENTS[type].schema.omit(["id", "type"])}
+      >
+        {({ handleReset, handleSubmit, isSubmitting, errors }) => (
+          <Stack>
+            {/*<BootstrapForm onReset={handleReset} onSubmit={handleSubmit}>*/}
```
Comment (markspolakovs) [inline scores-src/src/client/pages/CreateEventModal.tsx:51]: Be good to get rid of commented-out cruft

File: scores-src/src/common/formFields/index.tsx [hunk 4]
```diff
@@ -49,15 +59,27 @@ export function Field(props: FieldProps) {
...
+            <NumberInput
+              {...field}
+              value={field.value}
+              /* necessary to properly handle NumberInput onChange having a different param type */
+              onChange={(value) => form.setFieldValue(props.name, value)}
+              error={meta.touched && meta.error}
+              label={props.title}
+            />
+          ) : (
+            <TextInput
```
Comment (markspolakovs) [inline scores-src/src/common/formFields/index.tsx:74]: We need to display `props.helper` somewhere - it was handled by the FormWrapper earlier

File: scores-src/src/common/formFields/index.tsx [hunk 5]
```diff
@@ -22,23 +32,23 @@ interface FieldWrapperProps extends BaseFieldProps {
...
+//         <Form.Control.Feedback type="invalid">
+//           {meta.error}
+//         </Form.Control.Feedback>
+//       )}
+//       {props.helper && (
+//         <Form.Text className="text-muted">{props.helper}</Form.Text>
+//       )}
+//     </Form.Group>
+//   );
+// }
```
Comment (markspolakovs) [inline scores-src/src/common/formFields/index.tsx:52]: Die!

**Review by markspolakovs — CHANGES_REQUESTED** [review]
Almost there

**Review by markspolakovs — APPROVED** [review]

**Pull request merged**